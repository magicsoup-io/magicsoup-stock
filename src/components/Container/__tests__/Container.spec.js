import React from 'react'
import renderer from 'react-test-renderer'

import { Container } from '../../../index'

it('renders <Container /> without throwing', () => {
  const tree = renderer.create(<Container> I am a Container</Container>).toJSON()
  expect(tree).toMatchSnapshot()
})
