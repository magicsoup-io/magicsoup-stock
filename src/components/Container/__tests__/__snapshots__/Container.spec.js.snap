// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders <Container /> without throwing 1`] = `
<div
  className="sc-bdVaJa sc-bxivhb iPFsIz"
>
   I am a Container
</div>
`;
